src/core/src/package_managers/YumPackageManager.py

```diff
@@ -202,6 +202,10 @@ def is_package(chunk):
         lines = output.strip().split('\n')
 
         for line_index in range(0, len(lines)):
+            # Do not install Obsoleting Packages. The obsoleting packages list comes towards end in the output.
+            if lines[line_index].startswith("Obsoleting Packages"):
```

Comment:
trailing whitespaces are taken caere by startswith but use strip() to erase any leading whitespaces. (used in other places where lines[x] is used)

Reply:
Added strip().